enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -28,8 +30,8 @@
 (defn- strip-labels [path]
   (mapv #(dissoc % :label) path))
 
-(defn- random-key
-  ([prefix n] (random-key prefix n 0))
+(defn- random-keyword
+  ([prefix n] (random-keyword prefix n 0))
```

Comment:
I didn't mention this in my last review because I don't think it matters for SerDes but on looking at this again I thought I should bring it up just in case you didn't already know.

Keywords never get garbage collected so you should be careful when dynamically creating keywords. If you create a million one-off random keywords that are used just once, that's a million objects that will live in memory forever and never get GC'ed. 

Probably not important here since SerDes is a short-lived command but it's something to be mindful of in general. I think it might even warrant a comment acknowledging that, while explaining that it doesn't matter if we intern a bunch of dynamically generated keywords here since SerDes is a short-lived thing. Just in case someone gets nervous if they see it in the future.